### DISCUSSION TITLE
Should we prevent screenshotting on the app?

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#131] [Ideas & Features]
On Reddit, there are constant pictures of chats and profiles, often with people mocking them. While this can be funny at times, shouldn't people have privacy in their intimate conversations (as long as they aren't doing anything illegal or repugnant)? While it is not perfect, preventing the screenshotting of the application could be a simple way to reduce this kind of bullying.

**Comment by ECorreia45** [comment]
People can use other devices to take photos. This does not solve the problem.

**Reply by WilliamDormer** [reply]
I agree it doesn't remove it completely, but it would significantly reduce it, which is almost as valuable. Most people will not use a second device to take a photo just to repost it.

**Reply by ECorreia45** [reply]
If someone is trying to bully you, they will do whatever it takes.

**Comment by ajbott** [comment]
I think this would be a really good community control. I think a community should be able to enable this feature and to further prevent people taking screenshots, their profile ID would be displayed on the screen or some type of watermark to help prevent this. It wouldn't be perfect, but we may be able to find a way to prevent this all together. (maybe dynamically overlay their profile ID to the picture as well).

**Reply by ajbott** [reply]
To add onto this, we could incorporate within the report feature to say that your profile has been involved in an online posting. We could use this information to cross reference the people that have seen the profile to find the profile that took the screenshot. If there are multiple instances, then we could use that information to find people that constantly do this.
Would be much easier if they posted a message and then they reported and provided a link to the posting.
I don't think anything will be perfect since it is hard to link the in-app account to the social media account posting, but efforts can be made to make the app a safer place.
